### ISSUE TITLE
400 unknow error

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#175]
```
What steps will reproduce the problem?
1. Change the update interval to 1

What is the expected output? What do you see instead?
After a few updates, it stops to get the new twits and show a error on the
status bar: "400 unknow error"


What version of the product are you using? On what operating system?
ubuntu repository version

```

Original issue reported on code.google.com by `hboaventura` on 1 Apr 2010 at 12:24

**Comment by GoogleCodeExporter** [comment]
```
This is rate limit http://apiwiki.twitter.com/Rate-limiting
I will add a description for this error.
```

Original comment by `tro...@gmail.com` on 1 Apr 2010 at 1:24
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Yes, i read about it, but still, if I change de update interval to 1, this is 
about a
60 request por hour, almost half of what I can use
```

Original comment by `hboaventura` on 1 Apr 2010 at 1:34

**Comment by GoogleCodeExporter** [comment]
```
No, this is 60x3 requests per hour (home timeline, mentions, direct messages).
```

Original comment by `tro...@gmail.com` on 1 Apr 2010 at 3:08

**Comment by GoogleCodeExporter** [comment]
```
Ow, now I got it, probably is a good idea to up a little bit the update 
interval,
maybe 1.5 or 2
```

Original comment by `hboaventura` on 1 Apr 2010 at 3:49

**Comment by GoogleCodeExporter** [comment]
```
Closed by 4c04d9ce59 revision.
```

Original comment by `tro...@gmail.com` on 2 Apr 2010 at 7:36
- Changed state: **Fixed**
